src/App.js

```diff
@@ -1,13 +1,22 @@
 import React from 'react';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
 import Calculator from './Components/Calculator';
+import Home from './Components/Home';
+import Navbar from './Components/Navbar';
 import Quote from './Components/Quote';
 
 function App() {
   return (
-    <div>
-      <Calculator />
-      <Quote category="happiness" />
-    </div>
+    <BrowserRouter>
+      <div>
+        <Navbar />
+        <Routes>
+          <Route path="/" element={<Home />} />
+          <Route path="/Calculator" element={<Calculator />} />
+          <Route path="/Display" element={<Quote category="happiness" />} />
```

Comment:
[OPTIONAL] This is an optional but recommended kindly don't skip it, please use small letters for the path because its looks professional check below :point_down:  my suggestions :slightly_smiling_face: 

```suggestion
          <Route path="/calculator" element={<Calculator />} />
          <Route path="/display" element={<Quote category="happiness" />} />
```

- Also, change the path in navbar as well then it will be working fine :+1: